CONTRIBUTING.md

```diff
@@ -19,10 +21,25 @@ We'd love your help!
   - [Stable](#stable)
   - [Unmaintained](#unmaintained)
   - [Deprecated](#deprecated)
+- [Pull Request Merge Guidelines](#pull-request-merge-guidelines)
+  - [General Merge Requirements](#general-merge-requirements)
 - [Contributing Vendor Components](#contributing-vendor-components)
   - [Adding a New Vendor Component](#adding-a-new-vendor-component)
   - [Removing Vendor Components](#removing-vendor-components)
 
+## Development Quick Start
+
+To get the project started quickly, you can follow these steps. For more
+detailed instructions, see [development](#development) below.
+
+```sh
+git clone https://github.com/open-telemetry/opentelemetry-js-contrib.git
+cd opentelemetry-js-contrib
+npm install
```

Comment:
We could consider `npm ci` here. https://docs.npmjs.com/cli/v10/commands/npm-ci
It'll wipe node_modules dirs for a clean install. It'll error out if package-lock.json is out of sync.
I don't have a strong opinion (... but I feel like I should :)

Reply:
Oops good catch, I updated to `npm ci` in a different spot but missed this one. I agree, will update.

Reply:
Just came across the same question, I think the commit is still missing (possibly not yet pushed) :thinking:

Reply:
🤦‍♀️  I've been doing this lately pushing to a different branch. Should be updated now!

Reply:
yep now it's here, thx :slightly_smiling_face: